packages/cli/uni-builder/tests/__snapshots__/styledComponents.test.ts.snap

```diff
@@ -176,7 +178,7 @@ exports[`plugins/styled-components > should enable ssr when target contain node
               "<WORKSPACE>/node_modules/<PNPM_INNER>/babel-plugin-import/lib/index.js",
               {
                 "camel2DashComponentName": false,
-                "libraryDirectory": "lib",
+                "libraryDirectory": "es",
```

Comment:
should this be `lib` when target is node?

Reply:
This is the snapshot of target web. Previously, when the rsbuild target had node, all library directories used lib, but it seems to me that only node should use lib and web should use es.

<img width="1364" alt="image" src="https://github.com/user-attachments/assets/41143c6e-2d21-427f-a0ea-dd7042b10b7d">